src/main.js

```diff
@@ -1,31 +1,111 @@
-import {dataHomes} from "./data/data.js";
+const inputConditions = document.querySelector('.input-conditions');
+const modalConditions = document.querySelector('.main-form-modal');
+const downBtnAdults = document.querySelector('.adults-down');
+const upBtnAdults = document.querySelector('.adults-up');
+const countBtnAdults = document.querySelector('.adults');
+const countBtnRooms = document.querySelector('.rooms');
+const downBtnRooms = document.querySelector('.rooms-down');
+const upBtnRooms = document.querySelector('.rooms-up');
+const countBtnChilds = document.querySelector('.childs');
+const downBtnChilds = document.querySelector('.childs-down');
+const upBtnChilds = document.querySelector('.childs-up');
+const addChildInfo = document.querySelector('.modal-list-info');
 
-const homesCards = document.querySelector('.homes-cards');
+function createChildSelect() {
+  const extraChild = document.createElement('select');
+  extraChild.classList.add('modal-age-select');
+  extraChild.innerHTML = `
+  <option value="1" selected>0 years old</option>
+  <option value="2">1 year old</option>
+  <option value="3">2 years old</option>
+  <option value="4">3 years old</option>
+  <option value="5">4 years old</option>
+  <option value="6">5 years old</option>
+  <option value="7">6 years old</option>
+  <option value="8">7 years old</option>
+  <option value="9">8 years old</option>
+  <option value="10">9 years old</option>
+  <option value="11">10 years old</option>
+  <option value="12">11 years old</option>
+  <option value="13">12 years old</option>
+  <option value="14">13 years old</option>
+  <option value="15">14 years old</option>
+  <option value="16">15 years old</option>
+  <option value="17">16 years old</option>
+  <option value="18">17 years old</option>
+  `;
+  addChildInfo.append(extraChild);
+}
+
+function openModal() {
+  modalConditions.style.display = 'block';
+}
+function closeModal() {
+  modalConditions.style.display = 'none';
+}
+inputConditions.addEventListener('click', openModal);
+
+window.addEventListener('click', (event) => {
+  if (!event.target.contains('.main-form-modal')) {
```

Comment:
Please watch console when you are testing your code. This error clearly describe why you can't close a modal
<img width="1293" alt="Screenshot 2022-07-08 at 12 19 46" src="https://user-images.githubusercontent.com/25488411/177949459-7676d355-9b22-4d50-8a99-7cae184da142.png">
You should use node for contains function argument
```
  if (event.target.contains(modalConditions)) {
    closeModal();
  }
```

Reply:
Done.